transforms/helpers/util/types.ts

```diff
@@ -18,6 +18,7 @@ export function assert(
   message = 'Assertion Error'
 ): asserts condition {
   if (!condition) {
+    debugger;
```

Comment:
Do we want to keep this? I don't know the status of `console.assert`, but if it works, maybe it's
better long-term?

Reply:
Was going to revert this before merge as it was just a convenience during dev. When the error is thrown, it currently puts a stack trace in the output from the command.